Update es-419 localized messages

Gudahtt opened the pull request:
These translations were provided by Lionbridge.

Gudahtt committed:
Update es-419 localized messages

These translations were provided by Lionbridge

Gudahtt commented:
This is waiting on #10258, which conflicts with this and was submitted first.

Gudahtt commented:
This PR has been rebased onto `develop` to ensure that the `yourSigRequested` message was not present (it was removed in #10300).

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -6,7 +6,13 @@
   },
   "acceleratingATransaction": {
-    "message": "* Aumentar una transacción utilizando un precio de gas más alto aumenta sus posibilidades de ser procesada más rápido por la red, pero esto no siempre está garantizado."
+    "message": "* Usar un precio de combustible más alto para acelerar una transacción aumenta las posibilidades de un procesamiento más rápido en la red, pero esto no siempre se garantiza."
```
Previously we had decided to keep 'gas' in English as it's a bit of a technical term in Ethereum. 'Combustible' is a good translation, but is it confusing?

kumavis replied:
i think keeping "gas" is correct

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -21,34 +27,43 @@
   },
   "addToAddressBookModalPlaceholder": {
-    "message": "P. ej.: Juan García"
+    "message": "p. ej., John D."
```
Suggestion: "Fulano de Tal", Hispanic equivalent of John Doe.

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -57,118 +72,175 @@
   "appDescription": {
-    "message": "Una billetera de Ethereum en tu navegador",
-    "description": "The description of the application"
+    "message": "Una cartera de Ethereum en el explorador"
```
We are changing from 'billetera' to 'cartera' throughout this translation. That's a significant terminological change; I see no problem with comprehensibility, but others may have an opinion.

mapachurro replied:
```suggestion
    "message": "Una cartera de Ethereum en el navegador"
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -57,118 +72,175 @@
   },
   "attemptingConnect": {
-    "message": "Tratar de conectarse a blockchain."
+    "message": "Intentando una conexión a la cadena de bloques."
```
Are we good with a literal translation of 'blockchain', or do we want it in English?

kumavis replied:
my guess is "blockchain" is the word people will know

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -57,118 +72,175 @@
   },
   "browserNotSupported": {
-    "message": "Tu navegador no es compatible…"
+    "message": "El explorador no es compatible…"
```
```suggestion
    "message": "El navegador no es compatible…"
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -827,31 +1233,40 @@
   },
   "revealSeedWordsDescription": {
-    "message": "Si cambias de navegador o computadora, necesitarás esta frase de inicialización para acceder a tus cuentas. Guárdalas en un lugar seguro y secreto."
+    "message": "Si alguna vez cambia de explorador o de equipo, necesitará esta frase de recuperación para acceder a sus cuentas. Guárdela en un lugar seguro y secreto."
```
```suggestion
    "message": "Si alguna vez cambia de navegador o de equipo, necesitará esta frase de recuperación para acceder a sus cuentas. Guárdela en un lugar seguro y secreto."
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -1006,131 +1457,431 @@
   },
   "syncWithMobileDesc": {
-    "message": "Puedes sincronizar tus cuentas e información con tu dispositivo móvil. Abre la aplicación de MetaMask para dispositivos móviles, dirígete a \"Configuración\" y selecciona \"Sincronizar desde una extensión del navegador\""
+    "message": "Puede sincronizar sus cuentas y su información con el dispositivo móvil. Abra la aplicación móvil de MetaMask, vaya a \"Configuración\" y presione \"Sincronizar desde la extensión del explorador\""
```
```suggestion
    "message": "Puede sincronizar sus cuentas y su información con el dispositivo móvil. Abra la aplicación móvil de MetaMask, vaya a \"Configuración\" y presione \"Sincronizar desde la extensión del navegador\""
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -386,87 +608,125 @@
   },
   "gasPriceInfoTooltipContent": {
-    "message": "El precio del gas especifica la cantidad de Ethers que estás dispuesto a pagar por cada unidad de gas."
+    "message": "El precio de combustible especifica la cantidad de ethers que está dispuesto a pagar por cada unidad de combustible."
```
```suggestion
    "message": "El precio de combustible especifica la cantidad de ether que está dispuesto a pagar por cada unidad de combustible."
```

Gudahtt commented:
Hey @mapachurro , thanks for the review!

I just wanted to make sure you saw the first comment in this PR, about #10258. This PR is blocked until after that PR is merged, at which point we'll rebase this onto those changes. I'm sure most (or perhaps all) of your feedback here will end up being useful, but it might be worth holding off until then to do a thorough review because the set of changes will be different at that time.

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -528,184 +789,307 @@
+  },
   "metamaskDescription": {
-    "message": "Te estamos conectando a Ethereum y a la web descentralizada."
+    "message": "Conectándolo a Ethereum y a la Web descentralizada."
```
```suggestion
    "message": "Conectándole a Ethereum y a la Web descentralizada."
```

mapachurro replied:
I understand dialectical loismo dialectical concerns, but leismo allows gender-neutral expressions. Correcting throughout to avoid gendering the user as male.

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -528,184 +789,307 @@
   },
   "needEtherInWallet": {
-    "message": "Necesitarás tener Ethers en tu billetera para poder interactuar con aplicaciones descentralizadas a través de MetaMask."
+    "message": "Para interactuar con aplicaciones descentralizadas mediante MetaMask, necesitará ethers en su cartera."
```
```suggestion
    "message": "Para interactuar con aplicaciones descentralizadas mediante MetaMask, necesitará ether en su cartera."
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -528,184 +789,307 @@
   },
   "newToMetaMask": {
-    "message": "¿Nuevo en MetaMask?"
+    "message": "¿Es nuevo en MetaMask?"
```
We are gendering the user as male here. Suggest an alternative phrase such as "¿Recién llegó a MetaMask?" ('Recently arrived at MetaMask?')

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -943,59 +1376,77 @@
   },
   "sentEther": {
-    "message": "Ethers enviados"
+    "message": "enviar ether"
```
```suggestion
    "message": "Enviar ether"
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -1006,131 +1457,431 @@
   },
   "testFaucet": {
-    "message": "Grifo de prueba"
+    "message": "Probar faucet"
```
```suggestion
    "message": "Grifo de prueba"
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -1006,131 +1457,431 @@
   },
   "transactionCancelSuccess": {
-    "message": "La transacción se canceló con éxito en $2"
+    "message": "La transacción de canceló correctamente en $2"
```
```suggestion
    "message": "La transacción se canceló correctamente en $2"
```

mapachurro commented on app/_locales/es_419/messages.json:
```diff
@@ -1006,131 +1457,431 @@
   },
   "transactionConfirmed": {
-    "message": "Se confirmó la transacción en $2."
+    "message": "La transacción de confirmó en $2."
```
```suggestion
    "message": "La transacción se confirmó en $2."
```

mapachurro commented:
Hey, sorry--I was looking at #10272, and went to the latest part of the chain to add my comments. 

Overall it looks good--a very different tone than the previous version, more formal, less chatty, and some significant terminology changes. Most of the changes I've made here are just "cleaning it up".

> Hey @mapachurro , thanks for the review!
> 
> I just wanted to make sure you saw the first comment in this PR, about #10258. This PR is blocked until after that PR is merged, at which point we'll rebase this onto those changes. I'm sure most (or perhaps all) of your feedback here will end up being useful, but it might be worth holding off until then to do a thorough review because the set of changes will be different at that time.

kumavis committed:
Apply suggestions from code review

Co-authored-by: oliver renwick <oliver.renwick@gmail.com>

Gudahtt committed:
Lint fixes

The unused message `swapGetQuotes` has been removed, and the English
descriptions have been added for each message where present.

Gudahtt commented:
This PR has been rebased onto `develop`, and I've fixed any lint errors present.

Gudahtt commented:
This PR is now ready for review! It has been rebased onto `develop`, now that the other es-419 translation PR has been merged.

Note that this PR now consists of _alternative_ translations for messages that we already have translations for. These alternatives were written from scratch by translators without access to the translations on `develop` at the moment, so they may be worse than what we already have.

darkwing commented:
Hello @mapachurro, could we got another review on this when you get a moment?  Thank you!

Gudahtt commented:
I've just rebased this again to resolve conflicts (two messages were removed)

darkwing commented:
@mapachurro Would you have a moment for a quick review?

Gudahtt commented:
I've just rebased this again to resolve conflicts (a few messages were removed)

darkwing commented:
The localizers didn't have access to our current translations so we should likely provide those and have them be modified.  There's no guarantee that these are better than our current localizations.
